### DISCUSSION TITLE
Git pro

### DISCUSSION THREAD

**Discussion by coderstar1234** [community/community#49861] [Actions]
The are not my proof pro in GitHub.

**Comment by bbkx226** [comment]
What can I help you?

**Reply by coderstar1234** [reply]
Sir the cannot access pro service please help me...

**Reply by bbkx226** [reply]
You've to subscribe to the "Pro" plan or register as a student, through the student developer pack
FYI, here's the link you can refer to:
https://education.github.com/pack
https://docs.github.com/en/billing/managing-billing-for-your-github-account/upgrading-your-github-subscription

**Reply by coderstar1234** [reply]
Nice thanks

**Reply by bbkx226** [reply]
If you find this helpful, don't hesitate to mark my reply as answer
